bin/rails

```diff
@@ -1,9 +1,4 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path('../spring', __FILE__)
-rescue LoadError => e
-  raise unless e.message.include?('spring')
-end
```

Comment:
I think this will remove the spring functionality entirely. Did you try running `bin/spring stop` when you were debugging spring?  I've seen spring (often) get into a weird state that requires restarting it.

Reply:
Hmmm, I was assuming it was getting loaded some other way in Rails now, otherwise this wouldn’t have been removed. My symptom was that `rails <anything>` would just hang, but I don’t think I thought to try stopping spring.

Reply:
Yeah, rolling this back, everything seems to be fine now, so I guess this was just a transient Spring issue. >:(